helm/eirini/templates/bits.yaml

```diff
@@ -168,4 +185,4 @@ spec:
             backend:
               serviceName: bits
               servicePort: 8888
-{{- end }}
+{{- end }}
```

Comment:
Please add the newline back

Reply:
Thanks for the review. I have made the changes please check https://github.com/cloudfoundry-incubator/eirini-release/pull/69 request.